chainercv/datasets/cityscapes/cityscapes_test_image_dataset.py

```diff
@@ -0,0 +1,64 @@
+import glob
+import os
+
+from chainer import dataset
+from chainer.dataset import download
+from chainercv.utils import read_image
+
+
+class CityscapesTestImageDataset(dataset.DatasetMixin):
+
+    """Dataset class for test images of `Cityscapes dataset`_.
```

Comment:
```
Image dataset for test split of `Cityscapes dataset`_.
```

This is more consistent with other `Image dataset` (e.g. `ADE20KTestImageDataset`).